addons/website/static/src/snippets/s_text_cover/000.scss

```
@@ -2,7 +2,7 @@
     // Needed to be able to stretch the inner container so that
     // the snippet works with the 50% and 100% height
     > * {
-        &, > .row {
+        &:not(.o_scroll_button), > .row {
```

```suggestion
        > *:not(.o_scroll_button) {
           &, > .row {
```
Maybe more consistent because we don't need to target .o_scroll_button > .row.

---

Thanks for the suggestion, I have improved the code. If you have time to do a quick check, it would be nice